src/framework/preferences/js/StarterGrades.js

```diff
@@ -398,6 +452,7 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             textSize: "%messagePrefix/textSize.json",
             textFont: "%messagePrefix/textFont.json",
             lineSpace: "%messagePrefix/lineSpace.json",
+            blueColorFilter: "%messagePrefix/blueColorFilterjson",
```

Comment:
missing dot in from of "json". Please test to make sure messages are properly loaded and rendered. Thanks.